docs/gui.rst

```diff
@@ -74,39 +75,40 @@ Paint a window
 
     The data type of ``img`` must be one of:
 
-    * float32, clamped into [0, 1]
+    * ``float32``, range ``[0, 1]``
 
-    * float64, clamped into [0, 1]
+    * ``float64``, range ``[0, 1]``
 
-    * uint8, range [0, 255]
+    * ``uint8``, range ``[0, 255]``
 
-    * uint16, range [0, 65535]
+    * ``uint16``, range ``[0, 65535]``
 
-    * uint32, range [0, UINT_MAX]
+    * ``uint32``, range ``[0, 4294967295]``
 
 
 .. function:: gui.circle(pos, color = 0xFFFFFF, radius = 1)
 
     :parameter gui: (GUI) the window object
-    :parameter pos: (tuple of 2) the position of circle
-    :parameter color: (optional, RGB hex) color to fill the circle
-    :parameter radius: (optional, scalar) the radius of circle
+    :parameter pos: (tuple of 2) the position of the circle
+    :parameter color: (optional, RGB hex) the color to fill the circle
+    :parameter radius: (optional, scalar) the radius of the circle
```

Comment:
Double `the` sounds strange to me... I guess we mainly want to emphersize the uniqueness of that `circle`, instead of the property:
```suggestion
    :parameter pos: (tuple of 2) position of the circle
    :parameter color: (optional, RGB hex) the color to fill the circle
    :parameter radius: (optional, scalar) radius of the circle
```

Reply:
I second these changes. The more the articles the clearer the meaning, isn't it? Hahaha I'm just joking but here we need those for sure.